modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/itemselector/plugin.js

```diff
@@ -250,34 +248,15 @@
 						callback(imageSrc, selectedItem);
 					}
 					else {
-						var imageElement = new CKEDITOR.dom.element.createFromHtml(
-							'<img src="' + imageSrc + '">'
-						);
-
-						editor.insertElement(imageElement);
-
-						if (IE9AndLater) {
-							if (!editor.window.$.AlloyEditor) {
-								editor.insertHtml('&nbsp;');
-							}
-
-							var element = new CKEDITOR.dom.element('br');
-
-							editor.insertElement(element);
-							editor.getSelection();
+						var elementOuterHtml = '<img src="' + imageSrc + '">';
 
-							editor.fire('editorInteraction', {
-								nativeEvent: {},
-								selectionData: {
-									element,
-									region: element.getClientRect(),
-								},
-							});
-						}
-						else {
-							editor.execCommand('enter');
+						if (instance._isEmptySelection(editor)) {
+							var breakMarkup = '<br />';
+							elementOuterHtml = elementOuterHtml + breakMarkup;
```

Comment:
(Based on [your last comment](https://github.com/liferay-frontend/liferay-portal/pull/720#issuecomment-765397078) @georgel-pop-lr, not sure if this one is ready for review or not yet... but anyway) the `breakMarkup` temporary variable here doesn't seem to add much. You could even just do:

```js
elementOuterHTML += '<br />';
```

(Not a blocker, just wondering.)

Reply:
Hi @wincent,

Yes is ready for review, good point, I will push the change. 

Regards.

Reply:
hi all,

any news regarding this PR? 

Thanks

Reply:
@georgel-pop-lr   I guess we need to re-run ci tests or was that re-tested after the (force) push?(GitHub makes it look like we didn't)

Reply:
I rerun only source format.

I didn't rerun tests as the last change was only related to `elementOuterHTML += '<br />'; ` but I can do so if you think is needed.

Reply:
well, it's always a good idea even with small changes so please do when you can.